configure.ac

```diff
@@ -45,7 +45,7 @@ PKG_PROG_PKG_CONFIG
 
 AM_PATH_GLIB_2_0
 
-GIO_DEPENDENCY="gio-unix-2.0 >= 2.40.0 libgsystem >= 2015.1"
+GIO_DEPENDENCY="glib-2.0 >= 2.44 gio-unix-2.0 >= 2.40.0 libgsystem >= 2015.1"
```

Comment:
Also need to bump
`-DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_40 -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_40` in `Makefile.am`.